Qt/mainwindow.cpp

```diff
@@ -25,6 +25,8 @@ MainWindow::MainWindow(QWidget *parent, bool fullscreen) :
 	memoryTexWindow(0),
 	displaylistWindow(0)
 {
+    Q_INIT_RESOURCE(desktop_assets);
```

Comment:
nit: I recommend using `editorconfig`.  See here:

http://editorconfig.org/

We have a file that specifies the line ending / indentation styles we use.

-[Unknown]